recipe/meta.yaml

```diff
@@ -1,41 +1,72 @@
-{% set version = "2020.1" %}
+{% set version = "2020a" %}
 
 package:
   name: tzdata
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/t/tzdata/tzdata-{{ version }}.tar.gz
-  sha256: 5e0a01117b3dfc9be27ef727e8f39e355e29bbc788bcd8fe2110871df5f9a935
+  - url: https://data.iana.org/time-zones/releases/tzdata{{ version }}.tar.gz
+    sha256: 547161eca24d344e0b5f96aff6a76b454da295dc14ed4ca50c2355043fb899a2
+    patches:
+    - 0001-Add-TABDATA-to-install_data.patch
+  - url: https://data.iana.org/time-zones/releases/tzcode{{ version }}.tar.gz
+    sha256: 7d2af7120ee03df71fbca24031ccaf42404752e639196fe93c79a41b38a6d669
+    patches:
+    - 0001-Add-TABDATA-to-install_data.patch
 
 build:
   number: 0
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: generic
+  ignore_run_exports:
+    - libgcc-ng
 
 requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
This uses a compiler because it builds its own tools. We add the `ignore_run_exports` above to get a `noarch: generic` package without any other dependencies.